ui/src/app/Pages/employee/employee.component.html

```diff
@@ -95,9 +95,8 @@ <h1 fxLayoutAlign="center center">All Employees</h1>
             <mat-form-field appearance="outline" fxFlex.lt-lg="80%" fxFlex="60">
               <mat-label>Choose Company</mat-label>
               <mat-select #mode value="side" formControlName="companyId">
-                <mat-option value="side">Side</mat-option>
-                <mat-option value="over">Over</mat-option>
-                <mat-option value="push">Push</mat-option>
+                <mat-option value="Education">Education</mat-option>
+                <mat-option value="Government">Government</mat-option>
```

Comment:
use endpoint of company(get it from swagger) to show data in dropdown list 
don't make it static

![image](https://user-images.githubusercontent.com/96727804/200284204-84add2f6-3ba0-4185-84eb-8ac4504b38c9.png)